package.json

```diff
@@ -17,7 +17,7 @@
   "license": "Cisco EULA (https://www.cisco.com/c/en/us/products/end-user-license-agreement.html)",
   "author": "devsupport@webex.com",
   "engines": {
-    "node": ">=14.x <=16.x",
+    "node": "16.x",
```

Comment:
Should we make it `>=16.x`

Reply:
16 is end of life as of September 11, we should probabbly try a newer version? https://nodejs.org/en/blog/announcements/nodejs16-eol